samples/client/petstore/typescript-fetch/builds/oneOf/apis/DefaultApi.ts

```diff
@@ -51,4 +54,28 @@ export class DefaultApi extends runtime.BaseAPI {
         return await response.value();
     }
 
+    /**
+     */
+    async testArrayRaw(initOverrides?: RequestInit | runtime.InitOverrideFunction): Promise<runtime.ApiResponse<TestArrayResponse>> {
+        const queryParameters: any = {};
```

Comment:
Yes, in fact, it can even be [runtime.HTTPQuery](https://github.com/OpenAPITools/openapi-generator/blob/2dd6b47821ef5fd8a7aecea973f0db13aec66f42/modules/openapi-generator/src/main/resources/typescript-fetch/runtime.mustache#L282)

But i think this is outside of the scope of this PR as I did not make any changes to the api mustache, i just added a new testcase.
This PR is only about fixing the model mustache, not improving the api mustache, so I would recommend opening a new PR for this

Reply:
ah, sorry, I assumed because it was a newly generated block, it originated from your change, too.